server.js

```diff
@@ -16,7 +16,7 @@ app.use(
 );
 
 // Handle React routing, return all requests to React app
-app.get("*", (req, res) => {
+app.get("*", (request, res) => {
```

Comment:
Rename `res` -> `response` too, if you like to have similar convention like in L:7

Reply:
Yes I thought, I did that already..